package.json

```diff
@@ -28,40 +28,53 @@
-    "ci_": "lerna run --concurrency=1 --stream ci",
-    "clean": "npx lerna run --parallel clean",
+    "ci": "lerna run --ignore embark-dapp-* --stream ci",
+    "ci:dapps": "lerna run --concurrency=1 --scope embark-dapp-* --stream ci",
+    "ci:full": "npm-run-all cwtree \"ci -- --concurrency={1}\" ci:dapps cwtree -- 1",
+    "clean": "npx lerna run --stream clean",
+    "clean:full": "npx npm-run-all clean clean:top",
+    "clean:top": "npx rimraf node_modules",
     "cwtree": "node scripts/check-working-tree",
     "globalize": "node scripts/globalize",
     "lint": "lerna run --parallel lint",
     "package": "lerna run --parallel package",
-    "preqa": "yarn install && npm-run-all cwtree build:no-ui",
-    "qa": "npm-run-all cwtree reboot:full cwtree qa_ cwtree",
-    "qa_": "lerna run --concurrency=1 --stream qa",
-    "reboot": "npm run reset && yarn install",
-    "reboot:full": "npm run reset:full && yarn install",
+    "postclean": "npx lerna clean --yes",
+    "postreboot": "yarn install",
+    "postreboot:full": "yarn install",
+    "preci:full": "yarn install",
+    "preqa:full": "yarn install",
+    "qa": "lerna run --ignore embark-dapp-* --stream qa",
+    "qa:dapps": "lerna run --concurrency=1 --scope embark-dapp-* --stream qa",
+    "qa:full": "npm-run-all cwtree reboot:full cwtree \"qa -- --concurrency={1}\" qa:dapps cwtree -- 1",
+    "reboot": "npm run clean",
+    "reboot:full": "npm run clean:full",
     "release": "node scripts/release",
-    "reset": "npx lerna run --parallel reset",
-    "reset:full": "npx npm-run-all reset reset:top",
-    "reset:top": "npx rimraf node_modules",
+    "reset": "npx lerna run --stream reset",
     "start": "lerna run --parallel start",
-    "test": "lerna run --concurrency=1 --stream test",
-    "test:full": "npm-run-all lint typecheck test",
+    "test": "lerna run --ignore embark-dapp-* --stream test",
+    "test:dapps": "lerna run --concurrency=1 --scope embark-dapp-* --stream test",
+    "test:full": "npm-run-all test test:dapps",
     "typecheck": "lerna run --parallel typecheck",
     "watch": "lerna run --parallel watch"
   },
   "workspaces": {
     "packages": [
-      "packages/*",
-      "test_dapps/*"
+      "dapps/templates/*",
+      "dapps/tests/*",
+      "packages/*"
     ],
     "nohoist": [
+      "embark/embark-reset",
       "embark/embark-test-contract-0",
       "embark/embark-test-contract-1",
-      "test_app/embark-service",
-      "test_app/zeppelin-solidity",
-      "test_app/embarkjs-connector-web3",
-      "contracts_app/embarkjs-connector-web3"
+      "embark-dapp-template-boilerplate/embarkjs-connector-web3",
+      "embark-dapp-template-demo/embarkjs-connector-web3",
+      "embark-dapp-template-simple/embarkjs-connector-web3",
+      "embark-dapp-test-app/embark-dapp-test-service",
+      "embark-dapp-test-app/embarkjs-connector-web3",
+      "embark-dapp-test-app/zeppelin-solidity",
+      "embark-dapp-test-contracts/embarkjs-connector-web3",
```

Comment:
Are wildcards not supported here? Would make it easier to read if it's applicable.

Reply:
These have a special purpose and follow the convention described here:

https://yarnpkg.com/blog/2018/02/15/nohoist/

There are two pivotal tasks I added last week re: getting rid of most of them:

https://www.pivotaltracker.com/story/show/164275406
https://www.pivotaltracker.com/story/show/164275593